### PR TITLE
CIV-11256 claimant dashboard status

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-11256

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jarekPierchala — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -1206,14 +1206,16 @@
...
+        "CLAIM_ENDED": "Bu ichi stopio’r hawliad hwn",
+        "CLAIMANT_REQUESTED_CCJ": "Bu ichi ofyn am Ddyfarniad Llys Sirol yn erbyn {{ defendantName }}.",
         "NO_RESPONSE_ON_TIME": "Aros i'r diffynnydd ymateb",
         "NO_RESPONSE_ELIGIBLE_CCJ":  "Nid yw'r diffynnydd wedi ymateb i'ch hawliad. Gallwch ofyn am Ddyfarniad Llys Sirol yn ei erbyn.",
         "NOT_ADMITTED_CLAIMANT": "Ymateb i'r diffynnydd.",
         "REQUESTED_MORE_TIME_TO_RESPOND": "Mae {{defendantName}} wedi gofyn am fwy o amser i ymateb.",
         "RESPONSE_BY_POST": "Bydd yr hawliad yn parhau drwy'r post",
-        "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos"
+        "WAITING_COURT_REVIEW": "Aros i'r llys adolygu'r achos",
+        "ACCEPTED_PARTIAL_ADMISSION": "Rydych wedi derbyn addefiad rhannol y diffynnydd. Maent wedi dweud y byddant yn talu ar unwaith.",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/cy.json:1217]: Please, alphabetical order

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -1206,14 +1206,16 @@
...
+        "CLAIM_ENDED": "You stopped this claim",
+        "CLAIMANT_REQUESTED_CCJ": "You requested a County Court Judgment against {{defendantName}}.",
         "NO_RESPONSE_ON_TIME": "Wait for the defendant to respond.",
         "NO_RESPONSE_ELIGIBLE_CCJ":  "The defendant has not responded to your claim. You can request a County Court Judgment against them.",
         "NOT_ADMITTED_CLAIMANT": "Respond to the defendant.",
         "REQUESTED_MORE_TIME_TO_RESPOND": "{{defendantName}} has requested more time to respond.",
         "RESPONSE_BY_POST": "The claim will continue by post",
-        "WAITING_COURT_REVIEW": "Wait for the court to review the case"
+        "WAITING_COURT_REVIEW": "Wait for the court to review the case",
+        "ACCEPTED_PARTIAL_ADMISSION": "You have accepted the defendant’s part admission. They said they would pay immediately.",
```
Comment (portizdev) [inline src/main/modules/i18n/locales/en.json:1217]: Same here

**Review by jarekPierchala — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (ElifDenizEsea) [reply src/main/modules/i18n/locales/cy.json]: done

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (ElifDenizEsea) [reply src/main/modules/i18n/locales/en.json]: done

**Review by portizdev — DISMISSED** [review]

**Review by portizdev — APPROVED** [review]

**Review by manish14836 — DISMISSED** [review]

**Review by jeswanth-hmcts — DISMISSED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Review by jarekPierchala — DISMISSED** [review]

**Review by GorilaNaranja — DISMISSED** [review]

**Review by jeswanth-hmcts — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

**Pull request merged**